src/main/java/com/avengers/gamera/service/ArticleService.java

```diff
@@ -78,19 +78,20 @@ public ArticleGetDto createArticle(ArticlePostDto articlePostDto) {
         article.setAuthor(userService.findUser(articlePostDto.getAuthorId()));
         EArticleType articleType = articlePostDto.getType();
 
-        if (articleType == EArticleType.valueOf("REVIEW")) {
+        if (articleType == EArticleType.REVIEW) {
+            article.setGame(gameService.findActiveGame(articlePostDto.getGameId()));
+        } else if (articleType == EArticleType.NEWS && articlePostDto.getGameId() != null) {
             article.setGame(gameService.findActiveGame(articlePostDto.getGameId()));
```

Comment:
```suggestion
            article.setGame(Game.Builder().id(articlePostDto.getGameId()).build());
```
不需要查询一次, 直接放入id 会自动存的 article用了ManyToOne
不过你这里跟Nancy可能会改重复